CHANGELOG.md

```diff
@@ -50,6 +50,8 @@
 - Improved error messages when failing to parse a series of things.
 - A warning is now raised for unused binary operations, records, record access
   and record updates.
+- Improve error messages with a new `did_you_mean` algorithm; and added note
+  when trying to access custom types with multi-variant.
```

Comment:
This isn't right- this feature isn't in this PR.

Could you remove the reference to did_you_mean too as that's an implementation detail of the compiler and the changelog is written for users. We could just say that suggestions have been improved.

Reply:
Done. And thanks for the patience.
```
- Improved recommendations on error messages.
```